apis/keda/v1alpha1/triggerauthentication_types.go

```diff
@@ -297,6 +300,29 @@ type AzureKeyVaultCloudInfo struct {
 	ActiveDirectoryEndpoint string `json:"activeDirectoryEndpoint"`
 }
 
+type GCPSecretManager struct {
+	Secrets []GCPSecretManagerSecret `json:"secrets"`
+	// +optional
+	Credentials *GCPCredentials `json:"gcpCredentials"`
```

Comment:
nit: sorry, completely missed this yesterday, just to match how `AzureKeyVault` is implemented to keep the consistency
```suggestion
	Credentials *GCPCredentials `json:"credentials"`
```

Reply:
Agree!